.env.example

```
@@ -8,7 +8,11 @@ POSTGRES_PASSWORD=change-me-for-shared-environments
 # Backend application settings
 SAFEQUERY_APP_NAME=SafeQuery API
 SAFEQUERY_ENVIRONMENT=development
-SAFEQUERY_DATABASE_URL=postgresql://safequery:safequery@postgres:5432/safequery
+SAFEQUERY_APP_POSTGRES_URL=postgresql://safequery:safequery@postgres:5432/safequery
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SAFEQUERY_APP_POSTGRES_URL key should go before the SAFEQUERY_ENVIRONMENT key

(UnorderedKey)